src/components/elementView/detailedView/assetView/assetView.scss

```diff
@@ -34,17 +32,16 @@
 }
 
 .graph-container div {
-  // overflow: auto;
-  // background-color: red !important;
-  // width: 100% !important;
+
 
   display: block;
 }
 .graph-container svg {
   width: 100% !important;
 }
 .network-connections {
-  display: flex;
+  display: block;
```

Comment:
Does this make it so the graph isn't huge when there's no table?

Reply:
i made the graph show up below the table so its big but the same size between assets